operators/src/source/ogr_source.rs

```diff
@@ -496,42 +525,64 @@ where
 
         let mut dataset_options = DatasetOptions::default();
 
+        let headers = format!(
+            "HEADERS={}",
+            columns
+                .csv_headers
+                .as_ref()
+                .map_or("AUTO", CsvHeaders::option_value)
+        );
```

Comment:
You could put the `HEADERS=` into your to-string method (option_value).
Then you can just say `columns.csv_headers.unwrap_or(CsvHeaders::Auto).to_string()`.

I'm just looking for means to keep these huge methods as concise as possible :sweat_smile: .

Reply:
0fcb8508b434b9d310b9344204d08121dc741701